Source/WebKit/Shared/Cocoa/XPCEndpoint.h

```diff
@@ -30,6 +30,7 @@
 #include <WebKit/WKBase.h>
 #include <wtf/OSObjectPtr.h>
 #include <wtf/spi/darwin/XPCSPI.h>
+#include <wtf/text/ASCIILiteral.h>
```

Comment:
Can this use `Forward.h` instead?

Reply:
Give that there is this in the header:
```
static constexpr auto xpcMessageNameKey = "message-name"_s;
```

I don't think we can merely rely on forward declaration?

Reply:
OK